src/shapes/Circle.java

```diff
@@ -0,0 +1,18 @@
+package shapes;
+
+public class Circle implements Shape {
+
+
+    public Circle(int i) {
```

Comment:
You could rename the parameter as `radius`
for example instead of `public Circle(int i)`do `public Circle(int radius)`